### ISSUE TITLE
Lowercase the hell out of reposcript

### ISSUE THREAD

**Issue by Jegp** [repocad/reposcript#14]
Lowercase all input strings in the Tokenizer, so all functions, objects and variables will ignore case.

So a call to ``line(...)`` would be the same as ``LINE(...)`` or ``LiNe(...)``.

**Comment by oleegholm** [comment]
great! lets us dumb it down :)

**Comment by Jegp** [comment]
Closed in web

**Comment by Jegp** [comment]
I discovered that there's a bug in the name resolution, so a function 'x' clashes with a name 'x'. This gets worse by lowercasing everything, so I'll revert this until the other is fixed.

**Comment by Jegp** [comment]
Giving it a second thought; is it really a good idea? It's pretty obvious when a character is lowercase or uppercase and it gives some possibilities for better naming... Dunno. What do you think Ole?

**Comment by oleegholm** [comment]
I say no case sensitivity!

**Comment by oleegholm** [comment]
Lets try that anyway, keeping it simple..
